content/blog/go-sdk-road-to-2/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Modern Cloud Infrastructure in Golang  - The Road to 2.0"
+authors: ["evan-boyle"]
+tags: ["go", "golang", "aws", "gcp", "azure"]
```

Comment:
Do we need both a `go` and `golang` tag? (Our tags are getting busy!)

Reply:
JFWIW - I opened https://github.com/pulumi/docs/issues/2532 just now on the general issue here.